### ISSUE TITLE
[Markup and meta data] Page title tag isn't unique and/or meaningful. (09.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#67442]
#### VFS team name
Virtual Agent

#### VFS product name
Chatbot

#### VFS feature name
Prescription

#### Point of Contact/Reviewers
Erin White - @erinrwhite - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Page title tag isn't unique and/or meaningful.

### Issue Details
On the static web page, the first segment of the title tag "Virtual Agent" is the same as the URL slug but different from the H1 for the page "VA chatbot". 

### VA.gov Experience Standard
[Category Number 09, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.2_A

### Platform Recommendation
Align the H1, the first segment of the title tag, the current breadcrumb, and (if possible) URL to reflect the same page title.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by SarahKay8** [comment]
We have completed the [update page and title tag](https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/1210) based off of [feedback from CAIA last year](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67442) and have a [PR](https://github.com/department-of-veterans-affairs/content-build/pull/2299) that will change the title tag in the https://www.va.gov/contact-us/virtual-agent/ page from “Virtual Agent | Veteran Affairs” to “VA Chatbot | Veteran Affairs.” Inside the PR we included pictures of the html code that we changed as well as how the contact us page looks with the new change. There is no visual difference, but the title tag shown in the html code has changed.

Validated PR for reference:  https://github.com/department-of-veterans-affairs/content-build/pull/2299

cc: @lealmendozaalexander @nathalierayter

**Comment by SarahKay8** [comment]
Devs are still waiting on approval from cms-infrastructure or VA Platform CoP Frontend and will post on platform support so someone can review.

Change is not live in DEV until PR is complete 

@erinrwhite
